frontend/scripts/setup/ubuntu_setup.sh

```
@@ -0,0 +1,121 @@
+  if [ ! -d "$HOME/flutter" ]; then
+    if ! git clone https://github.com/flutter/flutter.git -b stable $HOME/flutter; then
+      echo_error "Failed to clone Flutter repository."
+      exit 1
+    fi
+  fi
+
+  echo_info "Adding Flutter to PATH..."
+  export PATH="$HOME/flutter/bin:$PATH"
+  if ! grep -q "flutter/bin" ~/.bashrc; then
+    echo "export PATH=\"$HOME/flutter/bin:\$PATH\"" >> ~/.bashrc
+    source ~/.bashrc
```

Shellcheck (0.10.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)